Samples/SVGViewer/SvgViewer.cs

```diff
@@ -69,15 +69,25 @@ private void TextBox1_KeyDown(object sender, KeyEventArgs e)
 
         private void RenderSvg(SvgDocument svgDoc)
         {
+#if NET5_0_OR_GREATER
+            if (OperatingSystem.IsWindows())
+                svgImage.Image?.Dispose();
+#else
             svgImage.Image?.Dispose();
+#endif
 
             //using (var render = new DebugRenderer())
             //    svgDoc.Draw(render);
             svgImage.Image = svgDoc.Draw();
 
             var baseUri = svgDoc.BaseUri;
             var outputDir = Path.GetDirectoryName(baseUri != null && baseUri.IsFile ? baseUri.LocalPath : Application.ExecutablePath);
+#if NET5_0_OR_GREATER
+            if (OperatingSystem.IsWindows())
+                svgImage.Image?.Save(Path.Combine(outputDir, "output.png"));

```

Comment:
So what woud happen on non-Windows platforms here? I'm not sure if we shall raise an exception, log a warning, or disable the whole function in this case, what do you think?

Reply:
SVGViewer is currently Windows platform only. (Because System.Drawing is Windows platform only...)

Here only resolve warning.
If we run SVGViewer on non-Windows platforms, we need to consider drawing system first.